task_2.py

```diff
@@ -0,0 +1,41 @@
+from utils import get_name_string, get_intro_string
```

Comment:
```suggestion
from utils import get_name_string, get_intro_string

print("""
....""")

TEMPLATE = 'Input {} value: '


def validate_non_zero(var_name):
    while True:
        var = get_input_value(var_name)
        if var != 0:
            return var
        print(f'{var_name} shouldn\'t be zero')


def get_input_value(input_variable):
    while True:
        try:
            return int(input(TEMPLATE.format(input_variable)))
        except ValueError as e:
            print(f'Error: {e}\n'
                  f'Input a valid number for {input_variable}')


x = get_input_value('x')
y = validate_non_zero('y')
z = get_input_value('z')

print(((15 - x / y) / (33 ** y + 19.3) + z))
```

Reply:
@vloooo , sorry, b.. idk, where is the mistake.. in `from utils import get_name_string, get_intro_string`?